src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -524,13 +546,26 @@ namespace webnn_native { namespace ie {
         DAWN_ASSERT(padding.size() == 4);
         std::vector<size_t> dilations(options->dilations,
                                       options->dilations + options->dilationsCount);
-        DAWN_ASSERT(strides.size() == 2);
+        DAWN_ASSERT(dilations.size() == 2);
+        std::vector<int32_t> outputPadding(options->outputPadding,
+                                           options->outputPadding + options->outputPaddingCount);
+        DAWN_ASSERT(outputPadding.size() == 2);
+        ngraph_node_t* outputShapeNode = nullptr;
+        if (options->outputSizes != nullptr) {
```

Comment:
It's better to be used in transpose case like
```
if (transpose && options->outputSizes != nullptr) {
}
```

Reply:
Optimized.